pkg/operator/operator.go

```diff
@@ -180,6 +191,20 @@ func (o *Operator) Start(stop <-chan struct{}) error {
 		log.Info("cache synced")
 	}
 
+	// Sync the CA cert every once in a while.
+	ticker := time.NewTicker(24 * time.Hour)
```

Comment:
This could be useful for rotation, but otherwise, what problem is it worth fixing in 24 hours that doesn't need to be fixed in <1 hour?

Reply:
What value do you recommend? Error retries, and (later) expanding status seem useful here.

Reply:
I redid this so that we just add a single event on startup to prime the controller. Additional edge triggering can come later.